frontend/lib/widgets/match_alert.dart

```diff
@@ -128,6 +131,21 @@ class MatchAlert extends StatelessWidget {
                 ],
               ),
             ),
+            SizedBox(
+              width: size.width * .8,
+              child: Expanded(
+                child: TextButton(
+                  onPressed: () {
+                    Nav.navigatorKey.currentState.popAndPushNamed("/chat-screen",
+                        arguments: ChatScreenArguments(
+                          isStudent ? object.id : AuthService().loggedAccountInfo.id,
+                          placement.employerId,
+                        ));
+                  },
+                  child: Text("Start a conversation"),
```

Comment:
something went wrong in case I'm a student and I want to start a conversation, It tells me that I'm not able to send a message

Reply:
@AliceCasali it should now be fixed. Could you please test it again and double check that it is correctly sending the message from the student side?